src/elm/Session/LoggedIn.elm

```diff
@@ -205,25 +197,15 @@ initModel shared authModel accountName selectedCommunity authToken =
     }
 
 
-type FeedbackStatus
-    = Success
-    | Failure
-
-
-type FeedbackVisibility
-    = Show FeedbackStatus String
-    | Hidden
-
-
 type ProfileStatus
     = Loading Eos.Name
     | LoadingFailed Eos.Name (Graphql.Http.Error (Maybe Profile.Model))
     | Loaded Profile.Model
 
 
-isAuth : Model -> Bool
-isAuth model =
-    Auth.isAuth model.auth
+hasPrivateKey : Model -> Bool
+hasPrivateKey model =
```

Comment:
so about this rename, does it make sense we consider if we have the private key? if we are logged we need to have it 100% sure. We will need the auth token and private key

Reply:
Correct me if I'm wrong, but from what I understand, we always have the private key **on localStorage**, but not necessarily in the `Model`. The `Model` only has access to the private key when:
- The user just logged in (inserted their 12 words and created a pin)
- We prompt the user for their pin. This happens when the user refreshes the page, so the `Model` is reset, and would need to get the private key from local storage, and we need the pin for that (if I understood it correctly)

So this function is asking if **the Model** has the private key. And yes, we always have the auth token in the `Model` **and** localStorage

Reply:
right! you are completely right, totally forgot about that! my bad